src/containerapp/azext_containerapp/_clients.py

```diff
@@ -1382,3 +1383,98 @@ def list(cls, cmd, resource_group_name, environment_name):
             dotNet_component_list.append(component)
 
         return dotNet_component_list
+
+
+class MaintenanceConfigPreviewClient():
+    api_version = "2024-10-02-preview"
+    maintenance_config_name = MAINTENANCE_CONFIG_DEFAULT_NAME
+
+    @classmethod
+    def show(cls, cmd, resource_group_name, environment_name):
+        management_hostname = cmd.cli_ctx.cloud.endpoints.resource_manager
+        sub_id = get_subscription_id(cmd.cli_ctx)
+        url_fmt = "{}/subscriptions/{}/resourceGroups/{}/providers/microsoft.app/managedenvironments/{}/maintenanceConfigurations/{}?api-version={}"
+        request_url = url_fmt.format(
+            management_hostname.strip('/'),
+            sub_id,
+            resource_group_name,
+            environment_name,
+            cls.maintenance_config_name,
+            cls.api_version)
+
+        r = send_raw_request(cmd.cli_ctx, "GET", request_url)
+
+        return r.json()
+
+    @classmethod
+    def add(cls, cmd, resource_group_name, environment_name, maintenance_config_envelope, no_wait=False):
```

Comment:
Recommend to use `create` as function name to consistent with other Client class.

Reply:
decided upon add/remove